.github/workflows/ci.yml

```diff
@@ -9,11 +9,33 @@ on:
 name: CI
 
 jobs:
+  php-lint:
+    name: PHP Lint
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        php-version: [ '5.3', '5.4', '5.5', '5.6', '7.0', '7.1', '7.2', '7.3', '7.4' ]
```

Comment:
Is there some way to declare the php version list only once in the file?

Reply:
No, none that I'm currently aware of. (And unfortunately, GitHub Actions do not support YAML anchors yet.)